dev/testing/6277/EdDSA.c

```diff
@@ -0,0 +1,210 @@
+#include <stdio.h>
```

Comment:
This directory needs to disappear, and be replaced by files sitting in the test folder.
Search for ecx, ed and X25519, X448 in the test folder..

Quite a few of the algorithm tests are driven by test/evp_test.c...
(See l/test/recipes/30-test_evp_data/evppkey_ecx.txt)

Reply:
this is temporary.  I just needed somewhere to track my dev test.

I will eventually move it to someplace sensible.